tests/antool/CMakeLists.txt

```diff
@@ -55,15 +56,20 @@ else()
 		endif()
 
 	else()
+		if(PMEMFILE_VLTRACE_BIN)
```

Comment:
Please use "option" function to describe this variable. Also don't use PMEMFILE_ prefix.